### ISSUE TITLE
dacpac extension - output during task execution

### ISSUE THREAD

**Issue by dzsquared** [microsoft/azuredatastudio#21332]
**Is your feature request related to a problem? Please describe.**
I might be missing it, but when I run a bacpac import or other operation using the dacpac extension - I can't find a way to see the trace/output generated by each step.

**Describe the solution or feature you'd like**
The first place I looked for it was in the "output" tab

**Comment by kisantia** [comment]
Correct, the output from the DacFx operation doesn't get output anywhere. Currently the extension just sends the request to STS, which kicks off the DacFx operation and only updates the task status after it succeeds or fails. 

If we want to output the status, we could try to hook up to the DiagnosticTrace of DacServices and have that output in an output channel, similar to build sql project. It might not be as straightforward though to continuously pass the status from DacServices in STS back to ADS, compared to how sql projects just calls dotnet build directly in ADS.

I remember before we were considering switching to use sqlpackage for these operations instead of calling DacServices in STS, but I don't remember why that was abandoned...
